applications/GeoMechanicsApplication/tests/cpp_tests/test_U_Pw_condition.cpp

```diff
@@ -0,0 +1,52 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+// Project includes
+#include "custom_conditions/U_Pw_condition.hpp"
+
+#include "geo_mechanics_fast_suite.h"
+
+using namespace Kratos;
+namespace Kratos::Testing {
+
+
+        /// <summary>
+        /// Tests the calculation of the left hand side matrix of the default UPwCondition, which should be an empty matrix.
+        /// </summary>
+        /// <param name=""></param>
+        /// <param name=""></param>
+        KRATOS_TEST_CASE_IN_SUITE(CalculateLeftHandSideUPwCondition, KratosGeoMechanicsFastSuite)
```

Comment:
I think it would be possible to capture the comment in the name of the test. Then we can remove the comment, which reduces a little bit of clutter.
```suggestion
        KRATOS_TEST_CASE_IN_SUITE(CalculateLeftHandSideUPwCondition_ReturnsEmptyMatrix, KratosGeoMechanicsFastSuite)
```